packages/react-ui-core/src/Gmap/Gmap.js

```diff
@@ -46,6 +46,9 @@ const MAP_CONTROLS = {
   mapTypeControl: true,
   scaleControl: true,
   rotateControl: true,
+  draggable: true,
```

Comment:
Are these `Gmap` changes necessary? I'm guessing they're true by default, since they were on before without this code.

Reply:
yes, we needed to add them to gmap to be able to change them for free draw

Reply:
Yup

Reply:
Not getting why we need them either..